docs/workflows/common-workflow-steps/firebase-app-distribution.md

```diff
@@ -0,0 +1,99 @@
+
+:::
+
+| Variable Name                   | Description                                                                                                                                                           | Status   |
+| ------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------- |
+| `$AC_FIREBASE_VERSION`         | Specifies the Firebase version to be used. Enter your Firebase version, such as `v11.11.0`, for a specific version. The default value is `latest`.                                                  | Required |
+| `$AC_FIREBASE_APP_PATH`        | Specifies the full path of the build. For example, `$AC_EXPORT_DIR/Myapp.ipa` or `$AC_APK_PATH`.                                                             | Required |
+| `$AC_FIREBASE_APP_ID`          | Specifies your app's Firebase App ID. You can find the app ID in the [Firebase console](https://console.firebase.google.com/u/0/).                                                                      | Required |
+| `$AC_FIREBASE_TOKEN`           | Specifies a refresh token that's printed when you authenticate with the `firebase login:ci` command. **Select either a Firebase token or a Google Service account**.  | Optional |
+| `$GOOGLE_APPLICATION_CREDENTIALS` | Specifies the path of the Google Service Account JSON. Upload the service account as a file to your environment group and name it `GOOGLE_APPLICATION_CREDENTIALS`. **Select either a Firebase token or a Google Service account**. | Optional |
+| `$AC_FIREBASE_RELEASE_NOTES`   | Specifies the release notes for this build. If you want to use a file for release notes, leave this field empty and configure the next section.                     | Optional |
+| `$AC_FIREBASE_RELEASE_NOTES_PATH` | If you use the Publish Release Notes component before this step, `release-notes.txt` will be used as release notes.                                                  | Optional |
+| `$AC_FIREBASE_GROUPS`          | Specifies the Firebase tester groups you want to invite.                                                                                               | Optional |
+| `$AC_FIREBASE_EXTRA_PARAMETERS` | Specifies extra command line parameters. Enter `--debug` for debug mode.                                                                                              | Optional |
+
+### Output Variables
+
+The **Firebase App Distribution** step does not produce any output variables. If the app is successfully distributed, the step will succeed. Otherwise, it will fail.
```

Comment:
```suggestion
The **Firebase App Distribution** step generates no output variables. The step succeeds if the app is distributed successfully; otherwise, it fails.
```